File: examples/scenarios/interrupted-flow-recovery/snippets/android/BootstrapConfig.kt
// Example placement: Application startup or feature bootstrap (pseudo-code).
// Adapt to: your app architecture, notification integration, and actual OpenClix runtime API names.
// Shared config: examples/scenarios/interrupted-flow-recovery/openclix.config.json
// Events used: replaceConfig
// Note: OpenClix APIs shown here are placeholders until the reference SDK is published.

fun loadInterruptedFlowRecoveryConfig(openClix: OpenClixManagerLike, configText: String) {
    // In a real app, read from assets/raw resources and parse into your config model.
    val configJson = parseJsonObject(configText)
    openClix.replaceConfig(configJson)
}

File: examples/scenarios/interrupted-flow-recovery/snippets/flutter/bootstrap_config.dart
// Example placement: app startup or feature bootstrap (pseudo-code).
// Adapt to: your app architecture, notification integration, and actual OpenClix runtime API names.
// Shared config: examples/scenarios/interrupted-flow-recovery/openclix.config.json
// Events used: replaceConfig
// Note: OpenClix APIs shown here are placeholders until the reference SDK is published.

void loadInterruptedFlowRecoveryConfig(OpenClixManagerLike openClix, String configJsonText) {
  // In a real app, read from bundled assets and decode JSON into your config object.
  final configJson = parseJsonObject(configJsonText);
  openClix.replaceConfig(configJson);
}
